### PR TITLE
Storing Flight and Holding Pattern added

### PR DESCRIPTION
**Pull request by JonasKaad**
Added support for the program to store the following in the database:
- Flights
- Airports
- Routes
- Holding Patterns

Tested with 5999 fallback flights with no errors or crashes and everything being stored correctly in the database.

firefox_BNvBlihD3R

9 Holding patterns found, stored in the database:

image

### CODE REVIEW

**Comment by JonasKaad** [comment]
Workaround: don't use EF core for this. 
Get the connection object from EF Core application context, and manually execute the insert statement. Then use the MySQL syntax `insert into ... on duplicate key ...` to make valid SQL Statements without errors.

Remember to use locks, as the code is run in parallel. 🔒🚀
- Alex

**Comment by AlexanderNorup** [comment]
Don't merge, as I want to test myself locally with the automated services.
But code looks fine. 
I'll merge when good.

**Review by AlexanderNorup — APPROVED** [review]
Tested against FF services.. Seems to work fine. Let's try it out.

**Pull request merged**
